src/include/ipc4/base_fw.h

```diff
@@ -288,6 +288,8 @@ enum ipc4_basefw_params {
 
 	/* Use LARGE_CONFIG_SET to change SDW ownership */
 	IPC4_SDW_OWNERSHIP = 31,
+
+	IPC4_SET_MIC_PRIVACY_FW_MANAGED_POLICY_MASK =36,
```

Comment:
There are quite big and enlightening comment about the earlier basefw_params. Would be good to ad something here too.